tooling/template-base/components/Signin/SigninPage.tsx

```diff
@@ -1,12 +1,29 @@
-import React, { useState } from 'react';
+import React, { useState, FormEvent } from 'react';
+import { gql, useMutation } from '@apollo/client';
 import Logo from '../Logo/Logo';
 
+const LoginDocument = gql`
+  mutation LoginMutation($email: String!, $password: String!) {
+    Login(email: $email, password: $password)
+  }
+`;
+
 const SigninPage = () => {
   const [userEmail, setUserEmail] = useState('');
   const [userPassword, setUserPassword] = useState('');
+  const [login] = useMutation(LoginDocument);
 
-  const signInHandler = e => {
-    console.log('Login Mutation Here');
+  const signInHandler = async (e: FormEvent) => {
+    e.preventDefault();
+    await login({
+      variables: { email: userEmail, password: userPassword }
+    })
+      .then(() => {
+        window.location.href = '/';
```

Comment:
Bit of a nitpick since users can change this to whatever location they want, but think we can copy existing functionality and redirect users to `/learn/` as seen here:

https://github.com/thoughtindustries/ti/blob/23ac2515750a0551bf419c5afac4bc46482641a0/lms/controllers/sessions.js#L76